recipes/cfgfile/0.2.8.2.1/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class CfgfileConan(ConanFile):
+    name = "cfgfile"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/igormironchik/cfgfile.git"
+    license = "MIT"
+    description = "Header-only library for reading/saving configuration files with schema defined in sources."
+    generators = "cmake"
+    topics = ("conan", "cfgfile", "configuration")
+    settings = "os", "arch", "compiler", "build_type"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "5",
+            "clang": "3.5",
+            "apple-clang": "10"
+        }
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
```

Comment:
```suggestion
        self._cmake = CMake(self)
        self._cmake.definitions["BUILD_EXAMPLES"] = False
        self._cmake.definitions["BUILD_TESTS"] = False
```

Do not build examples and test

Reply:
I don't build tests and examples. I use `generator/CMakeLists.txt` that doesn't build any examples and tests. With this item I totally disagree.

Reply:
Why don't you use the main cmake instead of walking to another cmake file? It would better using main and adding generator as optional, so `cmake --build . && cmake --build . --target install` is enough for all this

Reply:
Main CMake builds generator and use it as sub-folder, there I have options to build tests/examples. But for Conan recipe it's enough to use only generator's cmake. But if you want I can use main CMake and disable building of tests/examples with options.

Reply:
True, but remove the compiler from package id: 

```
def package_id(self):
    del self.info.settings.compiler
```

It doesn't affect validate() method.